client/asset/lcd/utils.go

```diff
@@ -191,6 +191,10 @@ func queryGateways(cliCtx context.CLIContext, cdc *codec.Codec, endpoint string)
 			return
 		}
 
+		if bytes.Equal(res, []byte("null")) {
+			res = []byte("[]")
+		}
```

Comment:
It's better to fix this by returning an empty array instead of null from the querier

Reply:
Array initialization missing